docs/source/commands/dev.mdx

```diff
@@ -60,6 +60,12 @@ If you don't pass the `--schema <PATH>` argument, `rover dev` will introspect th
 
 If you stop an attached `rover dev` process (by pressing `CTRL+C`), the supergraph will de-compose the removed subgraph and reload the router.
 
+### Advanced configuration
+
+To configure advanced router functionality like CORS settings or header passthrough for subgraphs, you can pass a valid [router configuration YAML file](/router/configuration/overview#yaml-config-file) to `rover dev` via the `--router-config <ROUTER_CONFIG_PATH>` argument.
```

Comment:
I think we only support a subset of those fields right now; should we mention that?

Reply:
The cloud router only supports some of these fields, but self hosted routers work with any config. Maybe we should include a warning if a config field is set that isn't supported by the cloud router?

I think this really demonstrates some product fragmentation we have between our cloud customers and our self-hosted OSS customers. I think we could make this a bit easier if we could differentiate between these types of use cases by configuration we manage in a `.apollo` directory.

Reply:
oh, yeah, wow, I wasn't even thinking about self-hosted OSS because my head's been in the cloud!